src/rf/requirements/workers.txt

```diff
@@ -1,2 +1,3 @@
 GDAL==1.10.0
 numpy==1.10.1
+exifread==2.1.2
```

Comment:
The service name referenced in each is different. I think it would make sense to consolidate the service names but I think that is well beyond the scope of this card. I'm in no way against installing the same dependencies on the worker and on the application server.